tests/test_playbooks/tasks/katello_sync.yml

```diff
@@ -22,6 +22,7 @@
     that:
       - not synchronous or ((( result.task.ended_at | replace(' UTC', '') | to_datetime ) - ( result.task.started_at | replace(' UTC', '') | to_datetime )).total_seconds() >= 2)
       - synchronous or not result.task.ended_at  # result.task.ended_at is not set if it matches result.task.started_at
+  when: result.task
```

Comment:
In `check_mode` we simply don't trigger a task, and cannot expect to analyse that one. I think this is OK.

Reply:
I think that's OK